homeassistant/components/ovo_energy/strings.json

```
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "error": {
+      "authorization_error": "Authorization error. Check your credentials.",
+      "connection_error": "Could not connect to OVO Energy."
+    },
+    "step": {
+      "user": {
+        "data": {
+          "username": "Username",
+          "password": "Password"
```

Use common keys, which instantly helps with other languages in Lokalise.

```suggestion
          "username": "[%key:common::config_flow::data::username%]",
          "password": "[%key:common::config_flow::data::password%]"
```